### ISSUE TITLE
Installation on Ubuntu

### ISSUE THREAD

**Issue by NealSmithCRGT** [johadi10/PostIt#27]
When I try to install via NPM on Ubuntu 16.04 several dependencies are noted that result in errors:

icdev_admin@ICDEV1:~/PostIt$ npm install
npm WARN ajv-keywords@3.1.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN karma-mocha-reporter@2.2.5 requires a peer of karma@>=0.13 but none is installed. You must install peer dependencies yourself.
npm WARN karma-webpack@1.8.1 requires a peer of webpack@^1.1.0 || ^2 || ^2.1.0-beta || ^2.2.0-rc but none is installed. You must install peer dependencies yourself.
npm WARN postit@1.0.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

Then when I try to start the app it fails to build.

icdev_admin@ICDEV1:~/PostIt$ npm start

> postit@1.0.0 start /home/icdev_admin/PostIt
> npm run build

npm ERR! missing script: build

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/icdev_admin/.npm/_logs/2018-03-28T15_47_22_002Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! postit@1.0.0 start: `npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the postit@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/icdev_admin/.npm/_logs/2018-03-28T15_47_22_024Z-debug.log

Is there something missing in the directions for installation?

Thanks

**Comment by johadi** [comment]
Hi @NealSmithCRGT Thanks for spotting this. The issue has been resolved and I have updated the Readme file to show how to set up the application on development and on production. Kindly check and follow the guide on the `readme.md` file. If the issue persists, let me know. Also , make sure you pull the latest changes and checkout to develop branch as it is the updated branch. Thanks!

**Comment by NealSmithCRGT** [comment]
Thank you.  Fix is good.


Neal Smith
Vice President, Digital Service Innovation Center

[cid:image002.jpg@01D2C8E5.43263890]

8255 Greensboro Drive Suite 500  Tysons Corner, VA 22102
571-766-2827 (o) | 703-587-0724 (m) | SalientCRGT.com<http://SalientCRGT.com>

On Mar 31, 2018, at 6:50 AM, Jimoh Hadi <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @NealSmithCRGT<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FNealSmithCRGT&data=01%7C01%7Cneal.smith%40salientcrgt.com%7C59428e9777b34e3805ef08d596f53cb8%7C6499e3f51d3e4f099ad352b9e2f67fb8%7C1&sdata=UUv6etaFhYnlBMbxsCPMpNsYSyUE8G7VYvAuxbHYsG0%3D&reserved=0> Thanks for spotting this. The issue has been resolved and I have updated the Readme file to show how to set up the application on development and on production. Kindly check and follow the guide on the readme.md file and if the issue persists, let me know. Also , make sure you pull the latest changes and checkout to develop branch as it is the updated branch. Thanks!
